go.mod

```diff
@@ -2,5 +2,7 @@ module github.com/terraform-providers/terraform-provider-http
 
 require (
 	github.com/hashicorp/hcl v1.0.0 // indirect
-	github.com/hashicorp/terraform v0.12.8
+	github.com/hashicorp/terraform-plugin-sdk v1.7.0
 )
+
+go 1.13
```

Comment:
Might need to bump the version of go in `.go-version`. Regret ever encoding that file across all the providers.... it does play an important role on TeamCity though for both tests and releases.